src/assets/i18n/en.json5

```diff
@@ -3041,8 +3041,21 @@
 
   "sorting.dc.title.DESC": "Title Descending",
 
-  "sorting.score.DESC": "Relevance",
+  "sorting.score.ASC": "Relevance Ascending",
 
+  "sorting.score.DESC": "Relevance Descending",
```

Comment:
I'd recommend we change the text of these to be:

`"sorting.score.ASC": "Least Relevant"`
`"sorting.score.DESC": "Most Relevant"`

For me, at least, it's difficult to determine what "Relevance Descending" means....so, it may just be easier to work them in a more understandable way.

Reply:
@tdonohue I've updated the suggested translations. I've kept a translation also for `search.results.head` since according to https://github.com/DSpace/dspace-angular/blob/8cade4fbf5635ff7fc2cce6e126ba008ebf30692/src/app/shared/search/search-results/search-results.component.html#L1

that label cannot be excluded a priori